### DISCUSSION TITLE
Quick.db store

### DISCUSSION THREAD

**Discussion by NoodlesDev987** [nfriedly/express-rate-limit#422] [Ideas]
How to make that rate limit data will be stored using quickdb ?

database methods here: https://www.npmjs.com/package/quick.db

**Comment by nfriedly** [comment]
Unfortunately, the API is not very well documented; this is something I'd like to improve in the future but don't have the time available right now.  However, you can look at the code for the memory store and the other stores and get a pretty good idea of what is needed.

**Comment by ririko5834** [comment]
Can you please create example code for quick.db and quick.mongo stores 🙏 ?

**Comment by nfriedly** [comment]
Here are some examples:
* https://github.com/wyattjoh/rate-limit-redis/blob/main/lib/redis-store.js
* https://github.com/linyows/rate-limit-memcached/blob/master/src/memcached-store.ts
* https://github.com/2do2go/rate-limit-mongo/blob/master/lib/mongoStore.js

That should be enough to get you going with the other stores you're hoping to use.
